service/src/main/java/org/apache/griffin/core/job/LivyTaskSubmitHelper.java

```diff
@@ -216,8 +222,105 @@ public void decreaseCurTaskNum(Long scheduleId) {
     private Map<String, Object> getResultByLivyId(Object livyBatchesId, TypeReference<HashMap<String, Object>> type)
             throws IOException {
         Map<String, Object> resultMap = new HashedMap();
-        String newResult = restTemplate.getForObject(uri + "/" + livyBatchesId, String.class);
-        return newResult == null ? resultMap : toEntity(newResult, type);
+        String livyUri = uri + "/" + livyBatchesId;
+        String result = getFromLivy(livyUri);
+        logger.info(result);
+        return result == null ? resultMap : toEntity(result, type);
+    }
+
+    public String postToLivy(String uri) {
+        logger.info("Post To Livy URI is: " + uri);
+        String needKerberos = env.getProperty("livy.need.kerberos");
+        logger.info("Need Kerberos:" + needKerberos);
+
+        HttpHeaders headers = new HttpHeaders();
+        headers.setContentType(MediaType.APPLICATION_JSON);
+        headers.set(REQUEST_BY_HEADER,"admin");
+
+        if (needKerberos == null || needKerberos.isEmpty()) {
+            logger.error("The property \"livy.need.kerberos\" is empty");
+            return null;
+        }
+
```

Comment:
If livy.need.kerberos is empty,  I think users may not need Kerberos. In other words, it's the same as  `livy.need.kerberos = false`.